app/components/category-members-grouped.js

```diff
@@ -30,7 +30,7 @@ export default Component.extend(
 
 				this.get('loadPage')(page)
 					.then(() => {
-						const navHeight = $('.site-head').outerHeight() + $('.site-head-fandom-bar').outerHeight(),
+						const navHeight = $('.site-head').outerHeight(),
```

Comment:
@kolek7 I think we should change it to check outerHeight of `.site-head-wrapper` to handle german wikis well

Reply:
BTW, we should also replace jquery here with plain javascript code, as we are getting rid of jquery in mobile-wiki at all